### DISCUSSION TITLE
Extension Version Question 8.0.1 installed.  Upgrade Column shows 8.2.1

### DISCUSSION THREAD

**Discussion by gnossosp** [DNNCommunity/Dnn.CommunityForums#863] [Q&A] [answered]
Using DNN v. 09.13.03 
Installed version 8.0.1 and having a few issues (won't mention here) but my extension bar shows an upgrade? of 8.2.1
I want to make sure my versions are latest before troubleshooting. 
![Clipboard_06-11-2024_01](https://github.com/DNNCommunity/Dnn.CommunityForums/assets/72934916/8dde33f7-f996-40cf-8fef-4b8551008ba0)
Is the Upgrade report premature OR is there a download somewhere else that I am not aware of? thanks

**Comment by johnhenley** [comment, marked as answer]
You are seeing these "fake" versions due to DNN telemetry picking up "news" of a version that really doesn't exist or hasn't been released. DNN has a "phone home" mechanism that sends versions for the platform and installed modules back to dnnplatform.io with the intention of collecting and corellating version information across the platform's ecosystem and sharing that information with the vendors and community. What sometimes happens though, is that a module developer might have DNN installed and be working on a module's repository (or fork of a module repository) and bumps the version in their internal build/version. But since they don't know about or perhaps forget to turn off this feature, their "update" makes its way into the telemetry, and gets pushed out to the ecosystem as a potential new version.

How do I know this? Because I've caused false updates to appear  as available because I test the community forums builds, and I forget to remove the option****. :)

More here:
https://dnncommunity.org/forums/using-dnn/extensions/core-modules/

8.0.1 is current version so if you have an issue please please post it.

Also, 8.1 is coming out soon, so your issue might (hopefully) already be addressed. 

https://github.com/DNNCommunity/Dnn.CommunityForums/releases/tag/08.01.00-RC1

**Comment by gnossosp** [comment]
JohnHenley- Thanks. I wanted to make sure I was using the most current version of DNN/ Forum Extension **before** trying to tackle any bugs.. since that is the FIRST question everyone needs to know!
